### ISSUE TITLE
Issues loading the data

### ISSUE THREAD

**Issue by ionymikler** [falcon-xu/LGViT#2]
Hi @falcon-xu !
I'm trying to run your pre-trained model, but I'm having issues with the data from huggingface. I get the following error:
`ValueError: BuilderConfig ParquetConfig(name='cifar100', version=0.0.0, data_dir=None, data_files={'train': ['cifar100/train-*'], 'test': ['cifar100/test-*']}, description=None, batch_size=None, columns=None, features=None) doesn't have a 'task' key`

Any idea why that might be?

**Comment by falcon-xu** [comment]
Hi, @ionymikler, 
Thank you for raising this issue!
The error might be related to the version of the `datasets` library or the way the dataset is being loaded. You can try the following code snippet to address the problem:
```python
from datasets import load_dataset, ImageClassification

# For CIFAR-100 dataset
task = ImageClassification(image_column='img', label_column='fine_label')
dataset = load_dataset(
    data_args.dataset_name,
    data_args.dataset_config_name,
    cache_dir=model_args.cache_dir,
    task=task,
    use_auth_token=True if model_args.use_auth_token else None,
    # ignore_verifications=True,
)
```
If you still encounter issues, please provide additional details such as:
- The exact code you're using to load the dataset.
- Versions of the datasets and transformers libraries.

Best regards,
Falcon

**Comment by ionymikler** [comment]
Hi!

I verified that the `datasets` version is 2.9.0, and indeed I was using 3.1.0.
When running with 2.9.0:
```python
from datasets import load_dataset, ImageClassification

task = ImageClassification(image_column='img', label_column='fine_label')
dataset = load_dataset("cifar100")
```
The `ImageClassification` class is found. **But** a new issue comes. In the `load_dataset` line I get:
```
ValueError                                Traceback (most recent call last)
Cell In[1], [line 4]
      [1] from datasets import load_dataset, ImageClassification
      [3] task = ImageClassification(image_column='img', label_column='fine_label')
----> [4] dataset = load_dataset("cifar100")
####
ValueError: Invalid pattern: '**' can only be an entire path component
```
Found [here](https://stackoverflow.com/questions/77671277/valueerror-invalid-pattern-can-only-be-an-entire-path-component) that the issue is with some library called `fsspec`. The suggested says it was solved for datasets==2.15.0, so I upgraded to it and the dataset load worked.

Bottom line I think the datasets library needs to be 2.15.0 or specify the fsspec version. I think the first is a better solution because moves the dataset lib forward.

**Comment by falcon-xu** [comment]
> Hi!
> 
> I verified that the `datasets` version is 2.9.0, and indeed I was using 3.1.0. When running with 2.9.0:
> 
> ```python
> from datasets import load_dataset, ImageClassification
> 
> task = ImageClassification(image_column='img', label_column='fine_label')
> dataset = load_dataset("cifar100")
> ```
> 
> The `ImageClassification` class is found. **But** a new issue comes. In the `load_dataset` line I get:
> 
> ```
> ValueError                                Traceback (most recent call last)
> Cell In[1], [line 4]
>       [1] from datasets import load_dataset, ImageClassification
>       [3] task = ImageClassification(image_column='img', label_column='fine_label')
> ----> [4] dataset = load_dataset("cifar100")
> ####
> ValueError: Invalid pattern: '**' can only be an entire path component
> ```
> 
> Found [here](https://stackoverflow.com/questions/77671277/valueerror-invalid-pattern-can-only-be-an-entire-path-component) that the issue is with some library called `fsspec`. The suggested says it was solved for datasets==2.15.0, so I upgraded to it and the dataset load worked.
> 
> Bottom line I think the datasets library needs to be 2.15.0 or specify the fsspec version. I think the first is a better solution because moves the dataset lib forward.

Hi, @ionymikler , thanks for your suggestion. 
We will update the required version of `datasets`.

Best regards,
Falcon
